src/components/game.tsx

```diff
@@ -72,40 +76,68 @@ export const Game = ({ userId: dbUserId }: GameProps) => {
         scene.current = this;
         // Creación del mapa
         const map = this.make.tilemap({ key: "tilemap" });
-        const tileset = map.addTilesetImage("tileset", "tiles")!;
+        // Cargar los tilesets según el nombre en el JSON
+        const tilesetAtlas = map.addTilesetImage("tileset_1", "atlas_48x");
+        const tilesetInteriors = map.addTilesetImage("tileset_2", "interiors");
+        const tilesetRoomBuilder = map.addTilesetImage("tileset_3", "room_builder");
+        // Puedes agregar más tilesets si tu tilemap.json los tiene
+        // Agrupa todos los tilesets en un array y filtra los null
+        const tilesets = [tilesetAtlas, tilesetInteriors, tilesetRoomBuilder].filter(Boolean) as Phaser.Tilemaps.Tileset[];
 
         // Creación de las capas del mapa
-        map.createLayer("Below Player", tileset, 0, 0)!;
-        map.createLayer("Carpets", tileset, 0, 0);
-        const wallLayer = map.createLayer("Wall", tileset, 0, 0)!;
-        const furnitureLayer = map.createLayer("Furniture", tileset, 0, 0)!;
-        const tablesLayer = map.createLayer("Tables", tileset, 0, 0)!;
-        const borderLayer = map.createLayer("Border", tileset, 0, 0)!;
-
-        wallLayer?.setCollisionByProperty({ collides: true });
-        furnitureLayer?.setCollisionByProperty({ collides: true });
-        tablesLayer?.setCollisionByProperty({ collides: true });
-        borderLayer?.setCollisionByProperty({ collides: true });
-
-        this.matter.world.convertTilemapLayer(wallLayer);
+        map.createLayer("Below Player", tilesets, 0, 0);
+        map.createLayer("Carpets", tilesets, 0, 0);
+
+        const floorLayer = map.createLayer("floor", tilesets, 0, 0)!;
```

Comment:
Ya que no usamos esta variable `floorLayer` después podemos dejarlo así para no tener errores de lint
```suggestion
        map.createLayer("floor", tilesets, 0, 0)!;
```